pkg/pgmodel/ingestor/duplicates.go

```diff
@@ -1,48 +0,0 @@
-// This file and its contents are licensed under the Apache License 2.0.
-// Please see the included NOTICE for copyright information and
-// LICENSE for a copy of the license.
-
-package ingestor
-
-import (
-	"sync"
-	"sync/atomic"
-	"time"
-
-	"github.com/prometheus/client_golang/prometheus"
-
-	"github.com/timescale/promscale/pkg/log"
-	"github.com/timescale/promscale/pkg/pgmodel/metrics"
-)
-
-const reportDuplicatesInterval = time.Minute
-
-var (
-	launchReporterOnce    sync.Once
-	duplicateMetricsTotal uint64
-)
-
-func init() {
-	atomic.StoreUint64(&duplicateMetricsTotal, 0)
-}
-
-func registerDuplicates(duplicateSamples int64) {
-	metrics.IngestorDuplicates.With(prometheus.Labels{"type": "metric", "kind": "sample"}).Add(float64(duplicateSamples))
-	metrics.IngestorDuplicates.With(prometheus.Labels{"type": "metric", "kind": "writes_to_db"}).Inc()
```

Comment:
I see that this series with `kind=writes_to_db` is not on the refactored code. Was it left out on purpose?

Reply:
Yes, it is just a duplicate of `kind="metrics"`.